README.md

```diff
@@ -24,7 +24,7 @@ Weather Forecastic allows users to track and predict their favorite cities' weat
 
 4. Go into this folder, then `conf/local/credentials.yml`
    - Fill in your datarobot and OpenAI credentials
-   - Note: Don't input the credentials from your notebook.
+   - Note: Notebook environment API keys are automatically recycled. It is recommended to use a DataRobot API Token from developer tools rather than from the notebook environment.
 
 5. Navigate to `conf/base/parameters.yml`
    - Add in the cities along with the weather features you want to collect
```

Comment:
I did not realize you could add additional predictors here. Since you have to change the logic in nodes.py and data_pull.ipynb what does adding additional variables do?